pennylane/data/data_manager/__init__.py

```diff
@@ -32,27 +33,12 @@
 from .foldermap import DataPath, FolderMapView, ParamArg
 from .params import DEFAULT, FULL, format_params
 
+GRAPHQL_URL = "https://cloud.pennylane.ai/graphql"
```

Comment:
```suggestion
import os

GRAPHQL_URL = os.getenv("DATASETS_ENDPOINT_URL", "https://cloud.pennylane.ai/graphql")
```
It'll be useful to be able to override this for testing new datasets